### PR TITLE
Baseline model

### PR DESCRIPTION
**Pull request by jorshi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
-# hear-baseline
-Simple baseline model for the HEAR 2021 NeurIPS competition
+![HEAR2021](https://neuralaudio.ai/assets/img/hear-header-sponsor.jpg)
+# HEAR 2021 Baseline
+
+A simple DSP-based audio embedding consisting of a Mel-frequency spectrogram followed
+by a random projection. Serves as the baseline model for the HEAR 2021 and implements
```
Comment (turian) [inline README.md:5]: naive baseline? Can we call the repo a naive baseline too?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+the [common API](https://neuralaudio.ai/hear2021-holistic-evaluation-of-audio-representations.html#common-api)
+required by the competition evaluation.
```
Comment (turian) [inline README.md:7]: Maybe add that a RP is less efficient than a CNN but is one of the simplest possible models code-wise.

Comment (turian) [reply README.md:7]: Also add a quick note how the weight file was generated

**Review by turian — APPROVED** [review]
Really nice. See the few comments above

File: README.md (hunk 1 above)
Comment (jorshi) [reply README.md:5]: Sure we can call this baseline the naive baseline. I'm a bit worried about the pip install / module name being too wordy. ```pip install hearnaivebasline`` is a bit awkward. What about this: if we create a conv baseline, which would be cool, can we include it in this repo? we could have `hearbaseline.naive` and `hearbaseline.conv`?

File: README.md (hunk 2 above)
Comment (jorshi) [reply README.md:7]: Sounds good

File: README.md [hunk 3]
```diff
@@ -1,2 +1,64 @@
...
+
+### Naive Baseline Model
+The naive baseline model produces log-scaled Mel-frequency spectrograms using a
+256-band Mel filter. Each frame of the spectrogram is then projected to 4096
+dimensions using a random projection matrix. Weights for the projection matrix were
+generated by sampling a normal distribution and are stored in this repository in the
+file `saved_models/naive_baseline.pt`.
+
+Using a random projection is less efficient
+than a CNN but is one of the simplest models to implement from a coding perspective.
```
Comment (jorshi) [inline README.md:37]: @turian -- added in this little section to address your comments above
